scripts manutencao/SCRIPT MIGRACAO_PENDENCIA_DIARIO_BORDO.sql

```diff
@@ -0,0 +1,37 @@
+do $$
+declare
+	contador integer := 0;
+	nroRegistros integer;
+	pendencia_db record;
+begin
+	 select into nroRegistros count(p.id) from pendencia p 
+		    inner join pendencia_aula pa on p.id = pa.pendencia_id 
+            inner join pendencia_usuario pu on pu.pendencia_id  = p.id 
+            inner join usuario u on u.id = pu.usuario_id 
+            where p.tipo = 9 and not p.excluido;
+		
+	for pendencia_db in 
+		select p.id as PendenciaId, pa.aula_id as AulaId, u.rf_codigo as ProfessorRf,
+			   p.criado_em as CriadoEm, p.criado_por as CriadoPor, p.criado_rf as CriadoRf, 
+			   p.alterado_em as AlteradoEm, p.alterado_por as AlteradoPor, p.alterado_rf as AlteradoRf
+            from pendencia p 
+            inner join pendencia_aula pa on p.id = pa.pendencia_id 
+            inner join pendencia_usuario pu on pu.pendencia_id  = p.id 
+            inner join usuario u on u.id = pu.usuario_id 
+            where p.tipo = 9 and not p.excluido
+  	loop
+  		insert into pendencia_diario_bordo (pendencia_id, aula_id, professor_rf, criado_em, 
```

Comment:
Na tabela **pendencia_diario_bordo** temos um campo **componente_curricular_id int8 NOT NULL,** e o mesmo não está sendo alimentado e está quebrando o script, por que ele é not null.

Reply:
corrigido